### ISSUE TITLE
数据类型的问题

### ISSUE THREAD

**Issue by JXHahahha** [MLMIP/MWG-Net#1]
你好，对于您的代码我现在存在一些问题，希望得到您的解答：
class DWT_block_1(nn.Module):
    def __init__(self, in_channels, out_channels):
        super(DWT_block_1, self).__init__()
        self.DWT = DWT_1()
        self.Conv = nn.Sequential(
            BasicConv2d(in_channels, out_channels, kernel_size=3, padding=1),
            BasicConv2d(out_channels, out_channels, kernel_size=3, padding=1), )
    def forward(self, x):
        x_LL, x_HL, x_LH, x_HH = self.DWT(x)        
        cat = torch.cat([x_LL, x_HL, x_LH], dim=1)       
        out = self.Conv(cat)
        return out
以上面这一段代码为例，在前想传播的过程中，你先通过小变换分别得到低频和高频子带： x_LL, x_HL, x_LH, x_HH,然后您接着对x_LL, x_HL, x_LH这三个小波信号进行了拼接，但是通过小波变换得到的 x_LL, x_HL, x_LH, x_HH的数据类型都是numpy.ndarray，但是对于torch.cat()中拼接的数据的数据类型必须是张量（tensor），但是在您代码里面这两行代码之间完全没有任何数据类型的转换，并且运行代码也存在错误，希望对于这个问题你可以给出回答。
